testing/evaluate_tests.py

```diff
@@ -113,9 +121,11 @@ def test_schema(resp, test_value, test_name=""):
         errs.update(res)
 
     if errs:
-        logger.info("Test {0} - Schema test failed for {1} with response {2}".format(
-            test_name, test_value, errs))
-        logger.info("Test {0} - Full response is {1}".format(test_name, resp))
+        logger.info(f"Test {test_name} - Schema test failed for {test_value} with response {errs}")
+        logger.error(test_name)
+        for key in errs.keys():
+            logger.error(f"Expected {key}: {test_value[key]}")
+            logger.error(f"Observed {key}: {errs[key]}\n")
```

Comment:
logging here now in `testing/evaluate_tests.py` since `test_value` and `errs` are dictionaries 

whereas in `test_nlp.py` the things in `test_results["test_failures"]` were strings (of dictionaries)